helpers/sendmail.js

```diff
@@ -0,0 +1,38 @@
+import nodemailer from 'nodemailer';
+import jwt from 'jsonwebtoken';
+import winston from 'winston';
+import mailtemplate from './mailtemplate';
+
+exports.sendEmail = async (userToBeVerified) => {
+  const token = jwt.sign(
+    { id: userToBeVerified.id },
+    process.env.SECRETE_KEY, { expiresIn: 60 * process.env.VERIFYTOKEN_EXPIRY }
+  );
+
+  // create template link
+  const templateLink = `${process.env.URL_HOST}${token}`;
+  const htmlTempate = mailtemplate.emailTemplate(templateLink);
+  const mailOptions = {
+    from: '"Authors haven" <noreply@authorhaven.sims.com>',
```

Comment:
Does this email work? Looks like an email that does not exist.

Reply:
it's a no reply email. The front end will indicate an email that can be replied to.

Reply:
I think that should come from an environment variable for now or fetch it from a database table.